moon/rospy/rospy_base.py

```diff
@@ -61,7 +62,7 @@ def setup_sockets(self, context=None):
         self.pull_socket.bind('tcp://*:' + self.PULL_PORT)
 
     def register_handlers(self):
-        pass
+        rospy.Subscriber('/clock', Clock, self.callback_clock)
```

Comment:
I guess the frequency is too high (for SubT it is 4ms step).

Reply:
I think tenth of a second precision (10Hz) would be sufficient for the above mentioned purposes. We can't do anything about ROS publishing frequency so not sure where it is too high, why it matters and what to do about it.

Reply:
`moon/rospy/rospy_base.py` is not related to SubT. If 10Hz is enough, I suggest to subsample the `/clock` topic to 10Hz right here, before sending it to `rosmsg.py`.